app/src/main/res/values/styles.xml

```diff
@@ -310,4 +310,10 @@
         <item name="android:textColor">?attr/colorAccent</item>
     </style>
 
+    <style name="FireDialog" parent="@style/Theme.Design.BottomSheetDialog">
+        <item name="android:windowIsFloating">false</item>
+        <item name="android:navigationBarColor">?toolbarBgColor</item>
```

Comment:
This produces a UI issue on low api devices. We had a similar issue in the past, where using white color makes all navigation buttons not visible. To avoid that bug I suggest  you to create a new style for `v27`.

Also, even if they are the same, I think is better to use `colorPrimary` instead.

Reply:
Oh, I didn't realize about that issue, sure, sounds good to me, thanks!

Reply:
No problem at all, it also happened to us in the past. Happy you can work on this and make it happen 🚀

Reply:
I removed `android:navigationBarColor` from the main style, and move it to `v27` file and now it's working great. I tried on `api 29 / 21 / 23`. I tried with `colorPrimary` but the result was `@color/almostBlackDark` instead of the actual primary color, I found another style on `v27` using `<item name="android:navigationBarColor">?toolbarBgColor</item>` so I think it's ok? Let me know if we can improve anything 🙌

Reply:
Also I noticed we can avoid the property `<item name="android:navigationBarColor">?toolbarBgColor</item>` and the `navigationBar` works fine, the nav buttons are visible, but the nav background is always black (I think it takes the navBar system default color)

Reply:
About using `?toolbarBgColor`, if we are already using that color is ok 👍 

And about about low api devices always being black is also ok, it's our expected behavior to avoid problems the problems I described with navigation buttons.